sdks/python/apache_beam/pipeline_test.py

```diff
@@ -132,6 +137,18 @@ def test_create_singleton_pcollection(self):
     assert_that(pcoll, equal_to([[1, 2, 3]]))
     pipeline.run()
 
+  @attr('ValidatesRunner')
+  def test_metrics_in_source(self):
+    pipeline = TestPipeline()
+    pcoll = pipeline | 'read' >> Read(FakeSource([1, 2, 3, 4, 5, 6]))
```

Comment:
Could you remove the `read` label? I do not think we should label transforms with the transform name.